### PR TITLE
Graphviz code for Fig 3-4 generation

### PR DESCRIPTION
**Pull request by ambientlight**
It would be great for readers to see how to generate annotated dependency tree in **Fig 3-4**.
This is also related to https://stackoverflow.com/a/44867616/2380455, where installation of dependencies has been presented for os x.

### CODE REVIEW

File: Chapter-3/dependency_parsing.py [hunk 1]
```diff
@@ -41,7 +41,14 @@
 dep_tree = [parse.tree() for parse in result][0]
 print dep_tree
 dep_tree.draw()
-                           
+
+'''
```
Comment (dipanjanS) [inline Chapter-3/dependency_parsing.py]: Can you uncomment this and send the pull request. If this is working, its ok to put the code snippet directly in the main file. No need to put it in quotes.

**Review by dipanjanS — CHANGES_REQUESTED** [review]
@ambientlight This looks interesting. Is it working with the data I had used in this book? Would be great if you could show a sample snapshot. Other than that please check out the review comment I posted in your code. Uncomment it out and remove the quotes. It is alright to directly put the code snippet in the file. Let me know once you do this, I will merge the PR.

**Comment by ambientlight** [comment]
>  Is it working with the data I had used in this book?

Sure here is the exact code I am using:

```
import os
import nltk
from nltk.parse.stanford import StanfordDependencyParser
from nltk.parse.stanford import StanfordParser

from spacy.en import English
from graphviz import Source

os.environ['CLASSPATH'] = r'/usr/local/Cellar/stanford-parser/3.6.0/libexec'
sentence = 'The brown fox is quick and he is jumping over the lazy dog'

# dependency tree printing
sdp = StanfordDependencyParser()
result = list(sdp.raw_parse(sentence))

dep_tree_dot_repr = [parse for parse in result][0].to_dot()
source = Source(dep_tree_dot_repr, filename="dep_tree", format="png")
source.view()
```

and the result:
dep_tree

File: Chapter-3/dependency_parsing.py (hunk 1 above)
Comment (ambientlight) [reply Chapter-3/dependency_parsing.py]: @dipanjanS that should be it

**Comment by dipanjanS** [comment]
Merged, thanks for the update!

**Pull request merged**